workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md

```diff
@@ -65,6 +65,37 @@ Complete the following tasks:
 
 ![](../.gitbook/assets/ex3/image004.png)
 
+* [ ] To complete the creation of the Cloudant service instance, accept the default value for region and keep the **Lite** price plan as the selected plan as shown in the following figure.
+
+![](../.gitbook/assets/ex3/image005.jpg)
+
+* [ ] Accept the default values for service name and resource group.
+* [ ] In **Available authentication methods**, select **Use only IAM**, as shown in the following figure. 
+
+![](../.gitbook/assets/ex3/image006.jpg)
+
+{% hint style="info" %}
+:memo: The Cloudant service has two authentication methods that are available: **Use both legacy credentials and IAM**, which means that both IAM and Legacy credentials can be used to access the account, and Use only IAM, which means that only IAM credentials are provided through service binding and credential generation. 
+* Legacy credentials enable login to Cloudant by using HTTP Basic authentication by providing a user name and password that is used for authentication. 
+* Identity and Access Management (IAM) provides a unified approach to managing user identities, services, and access control. IAM authentication requires that an IAM API key is exchanged for a time-limited access token before you can make a request to Cloudant. When the access token expires, the client must obtain a new one from the IAM token service.
+In this exercise, you use the IAM method as the authentication method for the Cloudant service.
+{% endhint %}
+
+* [ ] Click **Create**, as shown in the following figure.
+
+![](../.gitbook/assets/ex3/image007.jpg)
+
+* [ ] After the service is created, you are redirected to the Resource list window, as shown in the following figure. Open the Cloudant service instance overview by clicking the newly created service in the Resource list window. 
+
```

Comment:
This is a comment on the entire paragraph

Reply:
This is a replay to Don's comments on lines 89 - xx

Reply:
Done